src/basho_bench_stats_writer_csv.erl

```diff
@@ -88,6 +97,36 @@ report_error({_SummaryFile, ErrorsFile},
                io_lib:format("\"~w\",\"~w\"\n",
                              [Key, Count])).
 
+report_global_stats({Op,_}, Stats, Units) ->
+    %% Build up JSON structure representing statistices collected in folsom
+    P = proplists:get_value(percentile, Stats),
+    JsonElements0 = lists:foldl(fun(K, Acc) ->
+        case K of
+            %% Unpack percentiles list
+            percentile ->
+                [{p50, proplists:get_value(50, P)},
+                 {p75, proplists:get_value(75, P)},
+                 {p90, proplists:get_value(90, P)},
+                 {p95, proplists:get_value(95, P)},
+                 {p99, proplists:get_value(99, P)},
+                 {p999, proplists:get_value(999, P)}
+                 ] ++ Acc;
```

Comment:
This should be four space indentation and the first tuple should be indented with the rest. For example:

```
                [
                    {p50, proplists:get_value(50, P)},
                    {p75, proplists:get_value(75, P)},
                    {p90, proplists:get_value(90, P)},
                    {p95, proplists:get_value(95, P)},
                    {p99, proplists:get_value(99, P)},
                    {p999, proplists:get_value(999, P)}
                ] ++ Acc;
 ```